service_container/service_closures.rst

```diff
@@ -49,9 +49,15 @@ argument of type ``service_closure``:
             App\Service\MyService:
                 arguments: [!service_closure '@mailer']
 
+                # A shorthand is available
+                # arguments: ['@>mailer']
+
                 # In case the dependency is optional
                 # arguments: [!service_closure '@?mailer']
 
+                # Using the shorthand
+                # arguments: ['@>?mailer']
+
```

Comment:
We should add `versionadded` directive. Maybe after the last seealso

```
.. versionadded:: 7.3

    The ``@>`` shorthand was introduced in Symfony 7.3.
```

Reply:
Is this sufficient? I also reordered the examples so both shorthands are together.